src/pages/history/index.css

```diff
@@ -0,0 +1,21 @@
+.clearHistoryBtn {
+    position: absolute;
+    right: 1.5rem;
+    top: 1.5rem;
+    margin-bottom: 1rem;
+}
+
+.history-list-container {
+    display: grid;
+    grid-template-areas:
+      "menu btn btn btn btn btn"
+      "menu video video video video video"
+      
+      ;
+    grid-gap: 10px;
+  
+    grid-template-columns: 1fr 1fr 1fr 1fr 1fr 1fr;
```

Comment:
You can use `repeat` function here, like :
`grid-template-columns: repeat(6, 1fr)`